bos.workspace.json

```diff
@@ -0,0 +1,3 @@
+{
+  "apps": ["./apps/*"]
```

Comment:
This is correct -- but since this repo is going to be dedicated to only "embeds.near" account Id, we can make this just be a widget.

Do this:

move all of `apps/embeds`to:

`/widget`
`bos.config.json`
`package.json`

It's the app structure described [here](https://www.nearbuilders.org/docs.bos-workspace.near/widget/index?page=usage). Use [devs.near](https://github.com/NEARBuilders/devs.near) as a reference.


Then, add the mainnet and testnet github workflows. I'll correct them with public key and such -- do embeds.near in mainnet and embeds.testnet for testnet

Docs page: https://www.nearbuilders.org/docs.bos-workspace.near/widget/index?page=usage/deploy

Reply:
Resolved